utests/tests/ptf/test_mask.py

```diff
@@ -0,0 +1,39 @@
+from ptf.mask import Mask
+from ptf.packet_scapy import VXLAN, Ether, IP, TCP
+from ptf.testutils import simple_vxlan_packet
+
+
+class TestMask:
+    def test_mask__mask_simple_packet(self):
```

Comment:
I forgot to ask for the last PR, but is there a reason for the extra underscore in the nest name?

And for repeating `test_mask` in the method name when the class is already called `TestMask`?

Reply:
```text
test_{class name}__{test_title}
```

it's just some convention that I have adopted, we can change that, no problem.

Reply:
I don't have a strong preference, I was just curious if this was a widespread convention

Reply:
I am not quite sure if it is a widespread convention, but I like it, because, when someone sends you a test name that fails, you know where to find it :)